packages/components/badge/src/badge.scss

```diff
@@ -1,54 +1,124 @@
-@use 'sass:map';
-
-$variants: danger, success, info, warning, accent, neutral, primary;
-
 :host {
   align-items: center;
+  background: var(--sl-color-background-accent-grey-subtle);
   block-size: 1lh;
-  border-radius: var(--sl-border-radius-badge-full);
+  border-radius: var(--sl-size-borderRadius-full);
   box-sizing: border-box;
+  color: var(--sl-color-text-accent-grey-bold);
   display: inline-flex;
-  font: var(--sl-text-badge-text-md);
-  gap: var(--sl-space-badge-label-inline-md);
+  font-size: 0.857em;
+  gap: var(--sl-space-025);
+  line-height: 0.8lh;
   min-inline-size: 1lh;
-  padding-inline: var(--sl-space-badge-inline-md);
-  text-transform: var(--sl-text-case-badge-text-transform);
+  padding-inline: var(--sl-space-100);
+  text-transform: uppercase;
   vertical-align: middle;
 }
 
-@each $variant in $variants {
-  :host([variant='#{$variant}']) {
-    background: var(--sl-color-badge-subtle-#{$variant}-background);
-    color: var(--sl-color-badge-subtle-#{$variant}-foreground);
-  }
+:host([color='blue']),
+:host([variant='info']),
+:host([variant='primary']) {
+  background: var(--sl-color-background-accent-blue-subtle);
+  color: var(--sl-color-text-accent-blue-bold);
+}
 
-  :host([variant='#{$variant}'][emphasis='bold']) {
-    background: var(--sl-color-badge-bold-#{$variant}-background);
-    color: var(--sl-color-badge-bold-#{$variant}-foreground);
-  }
+:host([color='blue'][emphasis='bold']),
+:host([variant='primary'][emphasis='bold']) {
+  background: var(--sl-color-background-accent-blue-bold);
```

Comment:
There will be no background colours with transparency in the `bold` emphasis?

Reply:
No. No transparency for badges.